### DISCUSSION TITLE
unsupported device can't get it working

### DISCUSSION THREAD

**Discussion by kxqcvl** [Koenkk/zigbee2mqtt#15675] [Devices]
![image](https://user-images.githubusercontent.com/73292082/208836225-daaf24ca-f54d-4bff-94a3-59bd94cb9418.png)

I want to add the device : https://www.ecobright.be/zigbee-led-lamp-e27-globe-2000k-4000k-g125.html

and i think i followed the correct steps, my folder structure is like this : 

![image](https://user-images.githubusercontent.com/73292082/208836474-c9e5a334-83d8-4543-83ab-d84377655cdc.png)

with the lastest files from  https://github.com/Koenkk/zigbee-herdsman-converters

but it seems i'm doing something wrong, i've read the instructions over and over but seems i do not uderstand it, can someone lease tell me how to add this device please? I'm kinda a newbie at this but can't seem to get it working, is the folder structure right?

my config looks like this:

![image](https://user-images.githubusercontent.com/73292082/208836995-6d02b0c2-4ee0-41e0-b667-63ebacea2e28.png)

many thanks in advance :)

**Comment by Koenkk** [comment]
This device should work-out-of-the-box with z2m 1.28.4.
